### PR TITLE
added sign-in/sign-up pages

### PR DESCRIPTION
**Pull request by nikhiljugale007**
added signin-sign up pages

PREVIEW: https://deploy-preview-5--vercel-cart.netlify.app/pages/login.html

### CODE REVIEW

File: pages/auth.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	background-color: var(--hover-color);
+}
+.search-bar {
+	padding: 1rem;
+	border: 0;
+	border-radius: 8px;
+	background-color: var(--hover-color);
+}
+.search-bar:focus {
+	outline: none;
```
Comment (parvez-rashid) [inline pages/auth.css:45]: Avoid using `outline: none` as it is bad for accessibility.

File: pages/auth.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+.header-sub-container {
+	display: flex;
+	align-items: center;
+	gap: 0 2rem;
+}
+.search-container {
+	display: flex;
+	gap: 0;
+	background-color: var(--hover-color);
```
Comment (parvez-rashid) [inline pages/auth.css:36]: Give a more descriptive name for color like `--hover-color-grey`.

**Review by parvez-rashid — APPROVED** [review]
I've left my comments. All the best!

Edit: UI Review - The UI looks great, the login/signup sections are on-point. Great work!

File: pages/auth.css (hunk 2 above)
Comment (nikhiljugale007) [reply pages/auth.css:36]: Sure I will work on css variable naming more.

Comment (nikhiljugale007) [reply pages/auth.css:36]: But as of now, I cannot change, as I used this same name at a lot of places. But I will keep this suggestion in mind.

**Review by sri-rishi — APPROVED** [review]
Except changes mentioned above , Code and UI both looks good to me for merging

File: pages/auth.css (hunk 1 above)
Comment (nikhiljugale007) [reply pages/auth.css:45]: Will keep this in mind.

**Pull request merged**